test2.html

```diff
@@ -0,0 +1,206 @@
+        rel="stylesheet">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Concert+One&family=Days+One&family=Lemon&family=Montserrat+Alternates:wght@500&family=Shrikhand&family=Urbanist:wght@500&display=swap"
+        rel="stylesheet">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+    <div class="container">
+        <div class="space"></div>
+        <div class="row">
+
+            <div class="column" style="align-content: normal;text-align:right;">
+                <h1 class="title-header" style="font-family: Alfa Slab One;">hendrix today app</h1>
+                <p style="font-size: 25px;color:white"><em>stay up to date with Hendrix College</em></p>
+            </div>
+            <div class="column" style="align-content: center;text-align: left;">
+                <div class="rcorners">
+                    <img class="image" src="assets/img/ic_launcher.png" alt="App Screenshot" style="width:300px">
+                </div>
+                <div style="padding-left:40px;padding-top:10px">
+                    <a href="https://discotraystudios.github.io"
+                        style="color:white; font-size:20px; text-decoration:none; align-items: center; font-family:Concert One">
+                        a disco tray studios app
+                    </a>
+                </div>
+            </div>
+
+        </div>
+        
+        <div class="row" style="padding-top:20px">
+            <div class="column">
+                <div class="space"></div>
+            </div>
+            <div class="column">
+                <div id="icons">
+                    <a href="https://apps.apple.com/us/app/hendrix-today/id6455126560" target="_blank">
+                        <img src="assets/img/SVG/apple-badge.svg" alt="Screenshot" class="apple" width="200" />
+                    </a>
+                </div>
+
+            </div>
+            <div class="column">
+                <a href="https://play.google.com/store/apps/details?id=io.github.discotraystudios.good_vibes"
+                    target="_blank">
+                    <img src="assets/img/google-play-badge.png" alt="Screenshot" class="google" width="225" />
+                </a>
+            </div>
+            <div class="space"></div>
+
+        </div>
+        <div class="body-background">
+            <div class="small-space"></div>
+            <div class="row">
+                <div class="column" style="align-content:normal;text-align:right;">
+                    <h2 class="header" style="font-family: Concert One;">home page</h2>
+                    <p class="paragraph" style="margin-left:50%; font-family:Merriweather Sans;">Welcome to the new
+                        Hendrix Today app. Here you can find all the new and
```

Comment:
Let's add some exclamation points to these descriptions, because our app is awesome!